server/pension/tests.py

```diff
@@ -1,3 +1,8 @@
 from django.test import TestCase
 
-# Create your tests here.
+class TestPension(TestCase):
+    def test_index_view(self):
```

Comment:
@ranmoshe spacing above line 4 (the last line in the snippet that GitHub shows you, is the line that or mark).